client/redux/users.js

```diff
@@ -1,5 +1,5 @@
 import axios from 'axios';
-import { GET_USERS } from './actionConstants';
+import { GET_USERS, DESTROY_USER, UPDATE_USER } from './actionConstants';
 
```

Comment:
I think it might be bad to have a slice of the redux store dedicated to all users. Or at least we need to figure out a way to limit any users logic to admins only. You don't want non-admin users to get a list of all users.

Reply:
So should I add those to the redux user.js file?

Reply:
No, my thought was more like keeping it within the component and not using redux at all. Maybe there's a better approach, I don't know.